var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -11,11 +11,12 @@
 class Mvapich2(AutotoolsPackage):
     """MVAPICH2 is an MPI implementation for Infiniband networks."""
     homepage = "http://mvapich.cse.ohio-state.edu/"
-    url = "http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich2-2.2.tar.gz"
+    url = "http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich2-2.3.2.tar.gz"
     list_url = "http://mvapich.cse.ohio-state.edu/downloads/"
 
     # Prefer the latest stable release
-    version('2.3.1', sha256='314e12829f75f3ed83cd4779a972572d1787aac6543a3d024ea7c6080e0ee3bf', preferred=True)
+    version('2.3.2', sha256='30cc0d7bcaa075d204692f76bca4d65a539e0f661c7460ffa9f835d6249e1ebf', preferred=True)
```

Comment:
```suggestion
    version('2.3.2', sha256='30cc0d7bcaa075d204692f76bca4d65a539e0f661c7460ffa9f835d6249e1ebf')
```
This is not required, as Spack already defaults to the latest stable release by default.

Reply:
I have addressed this.

commit b2423e671c2d8f96e5016a5fbc15525b84fb1711
Author: Hari Subramoni <subramoni.1@osu.edu>
Date:   Mon Oct 28 23:50:27 2019 -0400

    Address review comment by Adam Stewart

Can this pull request be taken in?